src/mobile/apps/feature/routes.coffee

```diff
@@ -1,133 +1,9 @@
-_ = require 'underscore'
-Backbone = require 'backbone'
-accounting = require 'accounting'
-sd = require('sharify').data
 Feature = require '../../models/feature'
-Sale = require '../../models/sale'
-Artwork = require '../../models/artwork'
-{ API_URL } = require '../../config'
-sanitizeRedirect = require '../../components/sanitize_redirect'
-metaphysics = require '../../../lib/metaphysics'
-{ get } = require 'lodash'
 
 module.exports.index = (req, res, next) ->
   new Feature(id: req.params.id).fetch
     cache: true
     success: (feature) ->
       res.locals.sd.FEATURE = feature.toJSON()
       res.render 'page', feature: feature
-    error: res.backboneError
-
-module.exports.auctionRegister = (req, res, next) ->
-  # If not logged in, send them to sign up to return to this handler
```

Comment:
did we figure out where the `feature` app was used?

Reply:
I've found a few recent examples:

https://www.artsy.net/feature/discover-art-from-stuttgart
https://www.artsy.net/feature/laguna-art-museum-california-cool-benefit-auction-2020
https://www.artsy.net/feature/whitechapel-gallery-art-icon-benefit-auction-2020

From what I can tell, no features currently support registration or bidding directly on the feature page. Auctions have a corresponding `Feature` but the feature page `/feature/:feature_slug` now redirects you to the `auction/:auction_slug` route.